recipes/crisprme/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: crisprme
+  version: {{ version }}
+
+source:
+  url: https://github.com/samuelecancellieri/CRISPRme/archive/v{{ version }}.tar.gz
+  sha256: 427c755b0c489ded70bb7ada4b347302eab31aa51439c36e8ccf6375db3d4884
+  
+build:
+  number: 0
+  skip: True  # [osx or py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - libgomp  # [linux]
+  host:
+    - make
+    - boost
+    - python

```

Comment:
It looks like you just need a `run` section, without `build` or `host`. But then you should also specify `python >3.6` in the `run:` section.

Reply:
That'll be resolved by conda during installation, so there's no need for it here.